horace_core/sqw/coord_transform/@ortho_proj/private/projaxes_to_rlu_.m

```diff
@@ -1,18 +1,17 @@
-function [rlu_to_ustep, u_to_rlu, ulen, mess] = projaxes_to_rlu (proj, alatt, angdeg, ustep)
-% Determine matricies to convert rlu <=> projection axes, and their lengths
+function [rlu_to_ustep, u_to_rlu, ulen] = projaxes_to_rlu_(proj,ustep)
+% Determine matrices to convert rlu <=> projection axes, and the scaler
```

Comment:
typo: scaler->scalar
Unclear which scalar this is talking about. Is this an additional scalar displacement in addition to the matrix transformations?

Reply:
no -- its scale-r here. Not scalar certainly. Is is scalier?